operators/volume_loader/nrrd_loader.hpp

```diff
@@ -28,6 +29,11 @@ bool is_nrrd(const std::string& file_name);
 
 bool load_nrrd(const std::string& file_name, Volume& volume);
 
+std::vector<double> parse_vector(std::string str);
+
+void trim(std::string& s);
+
+std::vector<std::string> split_string_by_space(std::string str);
```

Comment:
These helper functions don't need to be exposed in the header.

Reply:
Thank you!